src/DssSpell.sol

```diff
@@ -55,172 +36,62 @@ contract DssSpellAction is DssAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
     //
-    uint256 constant THREE_PCT_RATE  = 1000000000937303470807876289;
+    uint256 constant TWO_PCT            = 1000000000627937192491029810;
+    uint256 constant THREE_PCT          = 1000000000937303470807876289;
+    uint256 constant THREE_PT_FIVE_PCT  = 1000000001090862085746321732;
+    uint256 constant FOUR_PCT           = 1000000001243680656318820312;
+    uint256 constant FOUR_PT_FIVE_PCT   = 1000000001395766281313196627;
+    uint256 constant FIVE_PCT           = 1000000001547125957863212448;
 
     uint256 constant MILLION    = 10**6;
     uint256 constant WAD        = 10**18;
     uint256 constant RAD        = 10**45;
 
-    address constant RWA001_OPERATOR           = 0x7709f0840097170E5cB1F8c890AcB8601d73b35f;
-    address constant RWA001_GEM                = 0x10b2aA5D77Aa6484886d8e244f0686aB319a270d;
-    address constant MCD_JOIN_RWA001_A         = 0x476b81c12Dc71EDfad1F64B9E07CaA60F4b156E2;
-    address constant RWA001_A_URN              = 0xa3342059BcDcFA57a13b12a35eD4BBE59B873005;
-    address constant RWA001_A_INPUT_CONDUIT    = 0x486C85e2bb9801d14f6A8fdb78F5108a0fd932f2;
-    address constant RWA001_A_OUTPUT_CONDUIT   = 0xb3eFb912e1cbC0B26FC17388Dd433Cecd2206C3d;
-    address constant MIP21_LIQUIDATION_ORACLE  = 0x88f88Bb9E66241B73B84f3A6E197FbBa487b1E30;
-    address constant SC_DOMAIN_DEPLOYER_07     = 0xDA0FaB0700A4389F6E6679aBAb1692B4601ce9bf;
+    // ETH-C
+    address constant MCD_JOIN_ETH_C = 0xF04a5cC80B1E94C69B48f5ee68a08CD2F09A7c3E;
```

Comment:
I noticed that this needed to be re-verified on etherscan. Did something change here?

Reply:
Yeah not sure about that either. I copy + pasted the exact code from the ETH-A gem join and it compiled into the same bytecode on Etherscan. Versions match.  Not sure what else it could be.

Reply:
ETH-B also needed to be re-verified?

Reply:
Ah, ok. I looked through it and it didn't look like there were any surprises. I've been deploying them right from DSS master branch fwiw.

Reply:
I verified this an hour ago in a diff.  The verified code was just a copy/past from `ETH-A` join including the
```
/**
 *Submitted for verification at Etherscan.io on 2019-11-14
*/
```

I didn't see any problem with it.  Did we just observe an etherscan blip?

Reply:
Yes accidentally copied the Etherscan inserted comment too. Oops.